st2common/st2common/transport/queues.py

```diff
@@ -53,53 +53,82 @@
-    'st2.actionrunner.pause', routing_key=action_constants.LIVEACTION_STATUS_PAUSING)
+    'st2.actionrunner.pause',
+    routing_key=action_constants.LIVEACTION_STATUS_PAUSING)
+
 ACTIONRUNNER_RESUME_QUEUE = liveaction.get_status_management_queue(
-    'st2.actionrunner.resume', routing_key=action_constants.LIVEACTION_STATUS_RESUMING)
+    'st2.actionrunner.resume',
+    routing_key=action_constants.LIVEACTION_STATUS_RESUMING)
+
 
 # Used by the exporter service
 EXPORTER_WORK_QUEUE = execution.get_queue(
-    'st2.exporter.work', routing_key=publishers.UPDATE_RK)
+    'st2.exporter.work',
+    routing_key=publishers.UPDATE_RK)
+
 
 # Used by the notifier service
-NOTIFIER_ACTIONUPDATE_WORK_QUEUE = execution.get_queue('st2.notifiers.execution.work',
-                                                       routing_key=publishers.UPDATE_RK)
+NOTIFIER_ACTIONUPDATE_WORK_QUEUE = execution.get_queue(
+    'st2.notifiers.execution.work',
+    routing_key=publishers.UPDATE_RK)
+
 
 # Used by the results tracker service
-RESULTSTRACKER_ACTIONSTATE_WORK_QUEUE = actionexecutionstate.get_queue('st2.resultstracker.work',
+RESULTSTRACKER_ACTIONSTATE_WORK_QUEUE = actionexecutionstate.get_queue(
+    'st2.resultstracker.work',
     routing_key=publishers.CREATE_RK)
 
+
 # Used by the rules engine service
 RULESENGINE_WORK_QUEUE = reactor.get_trigger_instances_queue(
-    name='st2.trigger_instances_dispatch.rules_engine', routing_key='#')
+    name='st2.trigger_instances_dispatch.rules_engine',
+    routing_key='#')
+
 
 # Used by the stream service
-STREAM_ANNOUNCEMENT_WORK_QUEUE = announcement.get_queue(routing_key=publishers.ANY_RK,
-                                                        exclusive=True,
-                                                        auto_delete=True)
-STREAM_EXECUTION_ALL_WORK_QUEUE = execution.get_queue(routing_key=publishers.ANY_RK,
-                                                      exclusive=True,
-                                                      auto_delete=True)
-STREAM_EXECUTION_UPDATE_WORK_QUEUE = execution.get_queue(routing_key=publishers.UPDATE_RK,
-                                                         exclusive=True,
-                                                         auto_delete=True)
-STREAM_LIVEACTION_WORK_QUEUE = Queue(None, liveaction.LIVEACTION_XCHG,
-                                     routing_key=publishers.ANY_RK,
-                                     exclusive=True, auto_delete=True)
-
-# TODO: Perhaps we should use pack.action name as routing key so we can do more efficient filtering
-# later, if needed
-STREAM_EXECUTION_OUTPUT_QUEUE = execution.get_output_queue(name=None,
-                                                           routing_key=publishers.CREATE_RK,
-                                                           exclusive=True,
-                                                           auto_delete=True)
+STREAM_ANNOUNCEMENT_WORK_QUEUE = announcement.get_queue(
```

Comment:
This is mostly trading having to scroll horizontally for having to scroll vertically. Does anybody actually develop on terminals that only support 80 characters across anymore? If not (and I certainly hope not), I think we need to adjust our allowed line lengths.